railties/lib/rails/commands/destroy/destroy_command.rb

```diff
@@ -14,6 +14,7 @@ def help
         end
       end
 
+      desc "destroy", "Destroys code generated with generate"
```

Comment:
This sounds a little extreme. :laughing:  Although "Destroys" matches the command, "Deletes" or "Removes" might be more natural?

```suggestion
      desc "destroy", "Removes code generated by `bin/rails generate`"
```